packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.types.ts

```diff
@@ -1,17 +1,45 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot, EventHandler, EventData } from '@fluentui/react-utilities';
+
+export type ColorAreaOnColorChangeData = EventData<'change', React.SyntheticEvent | MouseEvent> & {
+  color: HsvColor;
+};
 
 export type ColorAreaSlots = {
-  root: Slot<'div'>;
+  root: NonNullable<Slot<'div'>>;
+  thumb: NonNullable<Slot<'div'>>;
+  inputX: NonNullable<Slot<'input'>>;
+  inputY: NonNullable<Slot<'input'>>;
+};
+
+export type HsvColor = {
+  h: number;
+  s: number;
+  v: number;
+  a?: number;
 };
 
 /**
  * ColorArea Props
  */
-export type ColorAreaProps = ComponentProps<ColorAreaSlots> & {};
+export type ColorAreaProps = Omit<ComponentProps<Partial<ColorAreaSlots>>, 'onChange'> & {
```

Comment:
```suggestion
export type ColorAreaProps = Partial<ColorAreaSlots> & {
```

"div" does not have "onChange", so we can simplify